cmdutil/debug/debug.go

```diff
@@ -36,9 +40,10 @@ func New(l logrus.FieldLogger, port int) *Server {
 
 // Server wraps a gops server for easy use with oklog/group.
 type Server struct {
-	logger logrus.FieldLogger
-	addr   string
-	done   chan struct{}
+	logger      logrus.FieldLogger
+	addr        string
+	done        chan struct{}
+	pprofServer *http.Server
```

Comment:
I'd like to see this as a separate server. So something like `debu.PProfServer`.